### PR TITLE
UEFI-1234 : Test commit

### PR DESCRIPTION
**Pull request by rajiv448**
Please review the changes for UEFI-1234

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,6 +5,8 @@ export TOP_DIR = $(shell pwd)
 #export ARCH = arm
 #BOARD_NAME := zc702
 
+# This is test change for demo
```
Comment (syedsk) [inline Makefile:8]: Is this change really needed?

Comment (rajiv448) [reply Makefile:8]: Yes it is
